pkg/prowgen/prowgen_konflux.go

```diff
@@ -69,3 +101,155 @@ func GenerateKonflux(ctx context.Context, openshiftRelease Repository, configs [
 
 	return nil
 }
+
+func ServerlessOperatorKonfluxVersions(ctx context.Context) (map[string]string, error) {
+	r := Repository{Org: "openshift-knative", Repo: "serverless-operator"}
+	sortedBranches, err := Branches(ctx, r)
+	if err != nil {
+		return nil, fmt.Errorf("failed to list branches for %q: %w", r.RepositoryDirectory(), err)
+	}
+
+	config, err := LoadConfig("config/serverless-operator.yaml")
+	if err != nil {
+		return nil, fmt.Errorf("failed to load config for %q: %w", r.RepositoryDirectory(), err)
+	}
+
+	konfluxVersions := make(map[string]string, len(sortedBranches))
+	for i, branch := range sortedBranches {
+		// First branch we will build the operator with konflux
+		if v := os.Getenv("FIRST_KONFLUX_BRANCH"); (v != "" && v == branch) || branch == "release-1.35" {
```

Comment:
Does the FIRST_KONFLUX_BRANCH parameter make sense? We already know the first konflux branch will be 1.35. And if it gets delayed (for any reason) we can fix the constant in the code. I'm probably missing why it's an env variable. Thanks

Reply:
It helps with debugging and testing this flow